operator/src/main/resources/scripts/introspectDomain.py

```diff
@@ -475,10 +484,23 @@ def getClusteredServers(self, cluster):
   def addClusteredServer(self, cluster, server):
     name=self.name(server)
     self.writeln("- name: " + name)
-    self.writeln("  listenPort: " + str(server.getListenPort()))
+    if server.isListenPortEnabled():
+      self.writeln("  listenPort: " + str(server.getListenPort()))
     self.writeln("  listenAddress: " + self.quote(self.env.toDNS1123Legal(self.env.getDomainUID() + "-" + server.getName())))
+    if server.isAdministrationPortEnabled():
+      self.writeln("  adminPort: " + str(server.getAdministrationPort()))
+      self.writeln("  adminPortEnabled: " + self.booleanToString(server.isAdministrationPortEnabled()))
+    self.addSSL(server)
     self.addNetworkAccessPoints(server)
 
+  def addSSL(self, server):
+    ssl = server.getSSL()
+    if ssl is not None and ssl.isEnabled():
+      self.indent()
+      self.writeln("sslListenPort: " + str(ssl.getListenPort()))
+      self.writeln("sslPortEnabled: " + self.booleanToString(ssl.isEnabled()))
```

Comment:
Why pass 'sslPortEnabled'?  Isn't the presence of 'sslListenPort' enough to show that the port is enable?
Ditto for 'adminPortEnabled'.